lapis/job.py

```diff
@@ -23,6 +24,7 @@ class Job(object):
         "in_queue_until",
         "_name",
         "drone",
+        "fileprovider",
```

Comment:
Please attach the `fileprovider` to the drone. The job itself should still be very stupid and should only have connection to the *network* where files magically appear ;) This can go via drone.